shared/src/msi-installer/ContinuousProfiler/Files.wxs

```diff
@@ -4,25 +4,25 @@
      xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
   <?include $(sys.CURRENTDIR)\Config.wxi?>
   <Fragment>
-    <ComponentGroup Id="ContinuousProfiler.Files" Directory="ContinuousProfiler">
+    <ComponentGroup Id="ContinuousProfiler.Files" Directory="INSTALLFOLDER">
```

Comment:
What are the files that will be copied in this root folder for the profiler?
I'm not sure we have any since we got rid of Datadog.AutoInstrumentation.Profiler.Managed.dll

Reply:
You're right, there aren't any! The Tracer has this too, I'm not sure exactly why it's there, not sure if it's required or not. I'll try cleaning it up in a subsequent PR